TrackingTools/Configuration/python/TrackingTools_EventContent_cff.py

```diff
@@ -23,11 +23,11 @@
 )
 TrackingToolsRECO.outputCommands.extend(TrackingToolsAOD.outputCommands)
 _phase2_hgcal_TrackingRECO_tokeep = [
-        'keep recoGsfTracks_electronGsfTracksFromMultiCl_*_*',
-        'keep recoGsfTrackExtras_electronGsfTracksFromMultiCl_*_*',
-        'keep recoTrackExtras_electronGsfTracksFromMultiCl_*_*',
-        'keep TrackingRecHitsOwned_electronGsfTracksFromMultiCl_*_*',
-        'keep *_electronMergedSeedsFromMultiCl_*_*'
+        'keep recoGsfTracks_electronGsfTracks_*_*',
+        'keep recoGsfTrackExtras_electronGsfTracks_*_*',
+        'keep recoTrackExtras_electronGsfTracks_*_*',
+        'keep TrackingRecHitsOwned_electronGsfTracks_*_*',
+        'keep *_electronMergedSeeds_*_*'
 ]
 phase2_hgcal.toModify( TrackingToolsRECO,
                        outputCommands = TrackingToolsRECO.outputCommands + _phase2_hgcal_TrackingRECO_tokeep)
```

Comment:
this part is not needed anymore (everything is already in `TrackingToolsRECO`)

Reply:
Done.